odh/overlays/moc/monitoring/kustomization.yaml

```diff
@@ -8,3 +8,4 @@ resources:
   - ./servicemonitors
   - dashboards
   - datasource
+  - secrets
```

Comment:
Gotta put this under generators like so: 

```
---
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
namespace: opf-monitoring

resources:
  - ../../../base/monitoring
  - ./servicemonitors
  - dashboards
  - datasource
generators:
  - secrets/secret-generator.yaml
```

Reply:
ah yes, my bad..updated :+1: